apps/prairielearn/src/server.js

```diff
@@ -2165,7 +2158,7 @@ export async function insertDevUser() {
   await sqldb.queryAsync(adminSql, { user_id });
 }
 
-if (require.main === module && config.startServer) {
```

Comment:
Turns out this is required for our test suite 🙃 I'll add it back for now and figure out something else on the ESM conversion branch.

Reply:
Dang! So many hacks 😅